src/Ledger/Ratify.lagda

```diff
@@ -279,20 +470,62 @@ data _⊢_⇀⦇_,RATIFY'⦈_ : RatifyEnv → RatifyState → GovActionID × Gov
 
   -- remove expired actions
   -- NOTE: don't have to remove actions that can never be accpted because of sufficient no votes
+
   RATIFY-Reject : let open RatifyEnv Γ; st = proj₂ a in
     ¬ accepted Γ es st
     → expired currentEpoch st
     ────────────────────────────────
     Γ ⊢ ⟦ es , f , removed , d ⟧ʳ ⇀⦇ a ,RATIFY'⦈ ⟦ es , f , a ∷ removed , d ⟧ʳ
 
-  -- continue voting in the next epoch
+  -- Continue voting in the next epoch
+
   RATIFY-Continue : let open RatifyEnv Γ; st = proj₂ a; open GovActionState st in
     ¬ accepted Γ es st × ¬ expired currentEpoch st ⊎ delayed action prevAction es d
     ────────────────────────────────
     Γ ⊢ ⟦ es , f , removed , d ⟧ʳ ⇀⦇ a ,RATIFY'⦈ ⟦ es , a ∷ f , removed , d ⟧ʳ
 
-_⊢_⇀⦇_,RATIFY⦈_ : RatifyEnv → RatifyState → List (GovActionID × GovActionState) → RatifyState → Set
+_⊢_⇀⦇_,RATIFY⦈_ :  RatifyEnv → RatifyState → List (GovActionID × GovActionState)
+                   → RatifyState → Set
 _⊢_⇀⦇_,RATIFY⦈_ = SS⇒BS (λ where (Γ , _) → Γ ⊢_⇀⦇_,RATIFY'⦈_)
 \end{code}
-\caption{RATIFY transition system}
+} %% end small
+\caption{The RATIFY transition system}
+\label{fig:ratify-transition-system}
 \end{figure*}
+Figure~\ref{fig:ratify-transition-system} defines three rules, \RATIFYAccept, \RATIFYReject, and \RATIFYContinue, along with the relation \RATIFYsyntax.
+The latter is the transition relation for ratification of a \GovAction.  The three rules are briefly described here, followed by more details about how they work.
+\begin{itemize}
+  \item \RATIFYAccept asserts that the votes for a given \GovAction meets the threshold required for acceptance; the action is accepted and not delayed,
+  and \RATIFYAccept ratifies the action.
+  \item \RATIFYReject asserts that the given \GovAction is not \accepted and the \currentEpoch is \expired; it removes the governance action if it has expired.
+  \item \RATIFYContinue asserts that the number of votes in favor of a given \GovAction did not meet the threshold for acceptance and either the \currentEpoch is not \expired or the action is \delayed.
+\end{itemize}
+Finally, Figure~\ref{fig:ratify-transition-system} defines the relation \RATIFYsyntax.
+This is a wrapper around another function, represented by \SStoBS, which is a transformer of semantics from small-step to big-step.
+\begin{itemize}
```

Comment:
This & everything below seems like a leftover.

Reply:
Well, yes, maybe "left over" from the previous draft. However, I intentionally included these three points twice because I thought we could be help the reader by first briefly describing these three items, and then describe them again but in greater detail for those who want to know the specifics.  If you think that's overkill and I should simply delete the detailed descriptions, let me know.